src/index.ts

```diff
@@ -402,9 +419,31 @@ export class Loader {
     document.head.appendChild(script);
   }
 
-  private loadErrorCallback(e: Event): void {
-    this.onerrorEvent = e;
-    this.callback();
+  deleteScript(): void {
```

Comment:
While working on #102 I noticed this `deleteScript` function wasn't marked `private`. Was this intentional, is it a public facing API?